cirq/circuits/circuit_operation_test.py

```diff
@@ -542,6 +634,48 @@ def test_decompose_nested():
     assert cirq.Circuit(cirq.decompose(final_op)) == expected_circuit
 
 
+def test_decompose_repeated_nested_measurements():
```

Comment:
[No action required] Cool to see this realized in test form!

Reply:
😄